### PR TITLE
Add a notebook explaining recording and replaying in NanoVer

### PR DESCRIPTION
**Pull request by hjstroud**
This PR adds a Jupyter notebook to the `basics` directory that details how to record and replay simulations using the current in-built functionality of NanoVer. The aim is to get the user to be able to record their own trajectory and play it back in the same notebook.

Whilst writing this notebook I realised that the `openmm_polyalanine` notebook contained a TODO to add a notebook detailing how to use MDAnalysis with NanoVer recordings, which was resolved by PR #226, so this TODO was removed.

### CODE REVIEW

File: examples/basics/recording_and_replaying.ipynb [hunk 1]
```diff
@@ -0,0 +1,348 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Recording and replaying iMD trajectories with NanoVer\n",
+    "\n",
+    "Although the primary benefit of iMD-VR is the ability to simultaneously visualise and interact with molecular systems, it is often useful to save the trajectory produced in order to run analyses after the simulation is complete. This tutorial demonstrates how to use the in-built functionality of NanoVer to record trajectories from iMD simulations, and how to subsequently replay them\n",
```
Comment (rhoslynroebuck) [inline examples/basics/recording_and_replaying.ipynb:9]:
```suggestion
    "Although the primary benefit of iMD-VR is the ability to simultaneously visualise and interact with molecular systems, it is often useful to save the trajectory produced in order to run analyses after the simulation is complete. This tutorial demonstrates how to use the in-built functionality of NanoVer to record trajectories from iMD simulations, and how to subsequently replay them.\n",
```

File: examples/basics/recording_and_replaying.ipynb [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+    "As we are recording the data from this simulation, let's set the simulation to deliver the forces and velocities of the system in addition to the default data sent in the frames to the client. This can be achieved by setting the `include_forces` and `include_velocities` flags to `True`, as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Include forces and velocities\n",
```
Comment (rhoslynroebuck) [inline examples/basics/recording_and_replaying.ipynb:67]:
```suggestion
    "# [Optional] Include forces and velocities\n",
```

**Review by rhoslynroebuck — APPROVED** [review]
Yeah this looks good to me, I've added a couple of minor suggestions.

The only other thing is maybe adding a note at the beginning of the notebook directing users towards the [Recording page](https://irl2.github.io/nanover-docs/concepts/recording.html) in the docs. This page only has info about recording with the Rust server, but info about recording with the python server should be coming soon.

**Comment by hjstroud** [comment]
> Yeah this looks good to me, I've added a couple of minor suggestions.
> 
> The only other thing is maybe adding a note at the beginning of the notebook directing users towards the [Recording page](https://irl2.github.io/nanover-docs/concepts/recording.html) in the docs. This page only has info about recording with the Rust server, but info about recording with the python server should be coming soon.

This is a good suggestion, I have added a sentence at the start of the notebook to direct people to this page for more information on recording and playback with NanoVer.

**Pull request merged**